spec/lib/pdf_filler/nc_d400_pdf_spec.rb

```diff
@@ -43,36 +48,36 @@
           expect(pdf_fields['y_d400wf_fname1']).to eq 'North'
           expect(pdf_fields['y_d400wf_mi1']).to eq 'A'
           expect(pdf_fields['y_d400wf_lname1']).to eq 'Carolinianian'
-          expect(pdf_fields['y_d400wf_ssn1']).to eq '400000030'
-          expect(pdf_fields['y_d400wf_add']).to eq '123 Red Right Hand St Apt 1'
-          expect(pdf_fields['y_d400wf_apartment number']).to eq 'Apt 1'
-          expect(pdf_fields['y_d400wf_city']).to eq 'Raleigh'
+          expect(pdf_fields['y_d400wf_ssn1']).to eq '145004904'
+          expect(pdf_fields['y_d400wf_add']).to eq '7 Heavens Lane'
+          expect(pdf_fields['y_d400wf_apartment number']).to eq 'Apt 2'
+          expect(pdf_fields['y_d400wf_city']).to eq 'Cary'
           expect(pdf_fields['y_d400wf_state']).to eq 'NC'
-          expect(pdf_fields['y_d400wf_zip']).to eq '27513'
+          expect(pdf_fields['y_d400wf_zip']).to eq '27511'
 
           expect(pdf_fields['y_d400wf_fstat1']).to eq 'Yes'
           expect(pdf_fields['y_d400wf_fstat2']).to eq 'Off'
           expect(pdf_fields['y_d400wf_fstat3']).to eq 'Off'
           expect(pdf_fields['y_d400wf_fstat4']).to eq 'Off'
           expect(pdf_fields['y_d400wf_fstat5']).to eq 'Off'
 
-          expect(pdf_fields['y_d400wf_li6_good']).to eq '9000'
-          expect(pdf_fields['y_d400wf_li8_good']).to eq '9000'
+          expect(pdf_fields['y_d400wf_li6_good']).to eq '76902'
+          expect(pdf_fields['y_d400wf_li8_good']).to eq '76902'
```

Comment:
notes for self: both of these fields equal federal agi
`      <AdjustedGrossIncomeAmt>76902</AdjustedGrossIncomeAmt>`